airootfs/usr/local/bin/holoinstall

```diff
@@ -37,56 +37,91 @@ fi
 base_os_install() {
 	lsblk
 	read "?Enter your desired drive node here (for example, sda or nvme0n1): " DRIVEDEVICE
-	#read "?WARNING: This drive is going to be erased fully. Press enter to continue, or CTRL+Z to terminate"
 	
 	DEVICE="/dev/${DRIVEDEVICE}"
 	
 	INSTALLDEVICE="${DEVICE}"
+
+	echo "\n\nWARNING: The following drive is going to be erased fully. ALL DATA ON DRIVE $DEVICE WILL BE LOST!\n"
+	lsblk $DEVICE
+	echo -n "\nErase $DEVICE and begin installation(y/N): "
+	read ANS
+
+	if [[ ! ($ANS = 'n' || $ANS = 'N') ]]; then
+		echo "\nWiping partitions..."
+		sfdisk --delete ${DEVICE}
+		wipefs -a ${DEVICE}
+		echo "\nCreating new gpt partitions..."
+		parted ${DEVICE} mklabel gpt
+		destructive=true
+	fi
```

Comment:
Just wondering. What if user accidentally type other than n or N?

The whole disk got erased.

Reply:
Done